templates/zerver/help/message-a-stream-by-email.md

```diff
@@ -59,3 +59,11 @@ below by sending email to
   (e.g. a missed message email), a copy of the original message is
   automatically added to the bottom of your reply. By default, Zulip tries
   to remove that copied message. With this option, Zulip will include it.
+
+* **.prefer-html**: Email clients often send the message with two versions
+  of its content - plaintext and HTML. HTML version tends to carry all the
+  text formatting information. By default, Zulip will choose the plaintext,
+  but that can cause some of your formatting to be ignored. Use this option
+  to tell Zulip to use the HTML version. Keep in mind that converting HTML
+  to render the message with all the formatting is difficult, so this may
+  not always work as intended - letting Zulip use plaintext is safer.
```

Comment:
I'll want to do a tweak to simplify this to be good user-facing copy before merging; this is just a reminder to myself to do that.